src/Oro/Bundle/DistributionBundle/Command/ListAutoLoadingBundlesCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Oro\Bundle\DistributionBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Oro\Bundle\DistributionBundle\Console\Grid;
+
+class ListAutoLoadingBundlesCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('oro:auto-loading-bundles:list')
+            ->setDescription('List auto loading bundles by their priority');;
+    }
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Missing import of the Symfony\Component\Console\Output\OutputInterface namespace